### ISSUE TITLE
The accuracy of speechcommands transfer leanring  is too low

### ISSUE THREAD

**Issue by yunyunsan** [tensorflow/tfjs#3121]
When I used the code provided by https://codelabs.developers.google.com/codelabs/tensorflowjs-audio-codelab/index.html#0 to test, I found that the accuracy is too low and it is sensitive to any sound.
After that I added the following code to function buildModel (), but the problem is still not solved.
function buildModel() {
  model = tf.sequential();
  model.add(tf.layers.conv2d(
      {filters: 8, kernelSize: [2, 8],strides: [1, 1], activation: 'relu', inputShape: INPUT_SHAPE,}));
  model.add(tf.layers.maxPooling2d({poolSize: [2, 2], strides: [2, 2]}));
  model.add(
      tf.layers.conv2d({filters: 32, kernelSize: [2, 4],strides: [1, 1], activation: 'relu'}));
  model.add(tf.layers.maxPooling2d({poolSize: [2, 2], strides: [2, 2]}));
  model.add(
      tf.layers.conv2d({filters: 32, kernelSize: [2, 4], activation: 'relu'}));
  model.add(tf.layers.maxPooling2d({poolSize: [2, 2], strides: [2, 2]}));
  model.add(
      tf.layers.conv2d({filters: 32, kernelSize: [2, 4], activation: 'relu'}));
  model.add(tf.layers.maxPooling2d({poolSize: [2, 2], strides: [1, 2]}));
  model.add(tf.layers.flatten());
  model.add(tf.layers.dropout({rate: 0.25}));
  model.add(tf.layers.dense({units: 2000, activation: 'relu'}));
  model.add(tf.layers.dropout({rate: 0.5}));
  model.add(tf.layers.dense({units: 3, activation: 'softmax'}));
  const optimizer = tf.train.adam(0.01);
  model.compile({
   optimizer,
   loss: 'categoricalCrossentropy',
   metrics: ['accuracy']
 });
}
Or is my thinking wrong? So can I get some suggestions from here, I think that would be a great inspiration for my next direction.
Thank you very much and hope to get a reply.

**Comment by yunyunsan** [comment]
@tafsiri Thank you very much for your reply
